src/chains/compound/types.ts

```diff
@@ -95,12 +100,12 @@ export interface IVoteCast extends IEvent {
   kind: EventKind.VoteCast;
   voter: Address;
   id: number;
-  support: boolean;
+  support: number | boolean; // handle alpha and bravo support types
```

Comment:
I think it would be safe to marshal GovAlpha's bool into a number rather than using an OR type here.